python/dgl/data/graph_serialize.py

```diff
@@ -101,6 +101,9 @@ def save_graphs(filename, g_list, labels=None):
         The graphs to be saved.
     labels: dict[str, Tensor]
         labels should be dict of tensors, with str as keys
+    formats: str or list[str]
+        Save graph in specified formats. It could be any combination of
+        ``coo``, ``csc`` and ``csr``.

```

Comment:
Tell users its default behavior.

Reply:
How about this? :
```
Save graph in specified formats. It could be any combination of
        ``coo``, ``csc`` and ``csr``. If not specified, save one format
        only according to what format is available. If multiple formats
        are available, selection priority from high to low is ``coo``,
        ``csc``, ``csr``.
```